pom.xml

```diff
@@ -609,7 +615,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-war-plugin</artifactId>
-                <version>2.3</version>
+                <version>3.2.2</version>
```

Comment:
Wow, that's a big step forward. Any side effects to be expected?

Reply:
I would have to do more search if anyone had problem with new version, I didn't encountered any during war building.